api/routes/auth.js

```diff
@@ -208,6 +208,10 @@ router.post('/reset/:linkId', controllers.resetApprovementPassword);
 
 router.post('/reset-password', controllers.resetPassword);
 
-router.put('/change-password/:userId', controllers.changePassword);
+router.put(
+  '/change-password/:userId',
+  refreshToken,
```

Comment:
you can write it before any of route with imperative writing, check how middleware is working at express.

Reply:
Do you mean that it possible to use it like `app.use(refreshToken)`?

Reply:
> you can write it before any of route with imperative writing, check how middleware is working at express.

Do you mean that it possible to use it like app.use(refreshToken)?